README.md

```diff
@@ -4,18 +4,16 @@ Terraform module, which creates AWS Step Functions as well as required IAM role
 
 This Terraform module is the part of [serverless.tf framework](https://github.com/antonbabenko/serverless.tf), which aims to simplify all operations when working with the serverless in Terraform.
 
-
 ## Features
 
 - [x] Creates AWS Step Function
 - [x] Conditional creation for many types of resources
 - [x] Support IAM policy attachments for [Integrated Services (eg, Lambda, SQS, ECS, EKS, Batch, DynamoDB, etc)](https://docs.aws.amazon.com/step-functions/latest/dg/service-integration-iam-templates.html) and various ways to create and attach additional policies
 
-
 ## Usage
 
 ### Step Function
-  
+
```

Comment:
There is no need to touch README manually, run `pre-commit install` and `pre-commit run -a` to do required updates in the docs.